aries-site/src/layouts/main/Layout.js

```diff
@@ -12,13 +12,7 @@ const calcPad = size => {
   return val;
 };
 
-export const Layout = ({
-  children,
-  descriptiveHeader,
-  title,
-  isLanding,
-  isNavPage,
```

Comment:
We can remove the `isNavPage` prop from NavPage of `Guidelines`, `Foundation`, `Templates`, `Components`, `Design`, `Develop`, and `Resources`

Reply:
Music to my ears!